pkg/controller/datavolume-controller.go

```diff
@@ -53,10 +53,52 @@ const (
 	SuccessSynced = "Synced"
 	// ErrResourceExists provides a const to indicate a resource exists error
 	ErrResourceExists = "ErrResourceExists"
+	// ErrResourceDoesntExist provides a const to indicate a resource doesn't exist error
+	ErrResourceDoesntExist = "ErrResourceDoesntExist"
+	// ErrClaimLost provides a const to indicate a claim is lost
+	ErrClaimLost = "ErrClaimLost"
+	// DataVolumeFailed provides a const to represent DataVolume failed status
+	DataVolumeFailed = "DataVolumeFailed"
+	// ImportScheduled provides a const to indicate import is scheduled
+	ImportScheduled = "ImportScheduled"
+	// ImportInProgress provides a const to indicate an import is in progress
+	ImportInProgress = "ImportInProgress"
+	// ImportFailed provides a const to indicate import has failed
+	ImportFailed = "ImportFailed"
+	// ImportSucceeded provides a const to indicate import has succeeded
+	ImportSucceeded = "ImportSucceded"
+	// CloneScheduled provides a const to indicate clone is scheduled
+	CloneScheduled = "CloneScheduled"
+	// CloneInProgress provides a const to indicate clone is in progress
+	CloneInProgress = "CloneInProgress"
+	// CloneFailed provides a const to indicate clone has failed
+	CloneFailed = "CloneFailed"
+	// CloneSucceeded provides a const to indicate clone has succeeded
+	CloneSucceeded = "CloneSucceeded"
 	// MessageResourceExists provides a const to form a resource exists error message
 	MessageResourceExists = "Resource %q already exists and is not managed by DataVolume"
+	// MessageResourceDoesntExist provides a const to form a resource doesn't exist error message
+	MessageResourceDoesntExist = "Resource managed by %q doesn't exist"
 	// MessageResourceSynced provides a const to standardize a Resource Synced message
 	MessageResourceSynced = "DataVolume synced successfully"
+	// MessageErrClaimLost provides a const to form claim lost message
+	MessageErrClaimLost = "PVC %q lost"
+	// MessageImportScheduled provides a const to form import is scheduled message
+	MessageImportScheduled = "Import %q scheduled"
+	// MessageImportInProgress provides a const to form import is in progress message
+	MessageImportInProgress = "Import %q in progress"
+	// MessageImportFailed provides a const to form import has failed message
+	MessageImportFailed = "Failed to import PVC %q"
+	// MessageImportSucceeded provides a const to form import has succeeded message
+	MessageImportSucceeded = "Successfully imported PVC %q"
+	// MessageCloneScheduled provides a const to form clone is scheduled message
+	MessageCloneScheduled = "PVC %q cloning scheduled"
```

Comment:
Cloning is a bit of problem as we have 2 PVCs in potentially 2 different name spaces. However where we are generating the message it looks like we only have the target PVC available, with the target namespace (you can get it from the PVC) So I would change this to something like:
"Cloning from %s into %s/%s scheduled"
And then when you call printf pass source NS/PVC, namespace and, pvc name.
You can get the source NS/PVC from the AnnCloneOf ("k8s.io/CloneOf") annotation on the PVC